plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedViews.java

```diff
@@ -88,6 +90,19 @@ public void testShowTables()
         assertUpdate("DROP MATERIALIZED VIEW materialized_view_show_tables_test");
     }
 
+    @Test
+    public void testMaterializedViewsMetadataStorageCatalogSchemaName()
+    {
+        computeActual(format(
+                "CREATE MATERIALIZED VIEW test_materialized_view_%s AS " +
+                        "SELECT * FROM tpch.tiny.region LIMIT 1",
+                randomTableSuffix()));
+
+        assertQuery(
+                "SELECT storage_catalog, storage_schema FROM system.metadata.materialized_views",
```

Comment:
You can use the `getStorageTable` method in this class to get the expected name of storage table and use that to assert on the value of storage_table as well